packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -661,39 +765,175 @@ abstract class TableBase extends Resource implements ITable {
   }
 
   /**
-   * Metric for the user errors this table
+   * Metric for the user errors. Note that this metric reports user errors across all
+   * the tables in the account and region the table resides in.
    *
-   * @default sum over a minute
+   * By default, the metric will be calculated as a sum over a period of 5 minutes.
+   * You can customize this by using the `statistic` and `period` properties.
    */
   public metricUserErrors(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
-    return this.metric('UserErrors', { statistic: 'sum', ...props });
+
+    if (props?.dimensions) {
+      throw new Error("'dimensions' is not supported for the 'UserErrors' metric");
+    }
+
+    // overriding 'dimensions' here because this metric is an account metric.
+    // see 'UserErrors' in https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/metrics-dimensions.html
+    return this.metric('UserErrors', { statistic: 'sum', ...props, dimensions: {} });
   }
 
   /**
    * Metric for the conditional check failed requests this table
    *
-   * @default sum over a minute
+   * By default, the metric will be calculated as a sum over a period of 5 minutes.
+   * You can customize this by using the `statistic` and `period` properties.
    */
   public metricConditionalCheckFailedRequests(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
     return this.metric('ConditionalCheckFailedRequests', { statistic: 'sum', ...props });
   }
 
   /**
-   * Metric for the successful request latency this table
+   * Metric for the successful request latency this table.
    *
-   * @default avg over a minute
+   * @deprecated use `metricSuccessfulRequestLatencyForOperations`
    */
   public metricSuccessfulRequestLatency(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+
+    if (!props?.dimensions?.Operation || !props?.dimensions?.TableName) {
+      // Operation must be passed because its an operational metric.
+      // TableName must be passed because the dimensions here will override the default ones which contain the table name.
+      throw new Error("Both 'Operation' and 'TableName' dimensions must be passed for the 'SuccessfulRequestLatency' metric.");
+    }
+
     return this.metric('SuccessfulRequestLatency', { statistic: 'avg', ...props });
   }
 
+  /**
+   * Metric for the system errors this table.
+   *
+   * This will sum errors across all possible operations.
+   * Note that by default, each individual metric will be calculated as a sum over a period of 5 minutes.
+   * You can customize this by using the `statistic` and `period` properties.
+   */
+  public metricSystemErrorsForOperations(props?: SystemErrorsForOperationsMetricOptions): cloudwatch.IMetric {
+
+    if (props?.dimensions?.Operation) {
+      throw new Error("The Operation dimension is not supported. Use the 'operations' property.");
+    }
+
+    const operations = props?.operations ?? Object.values(Operation);
```

Comment:
`Object.values(Operation)` => are you sure this works? I was under the impression that TS enums rendered to bidirectional maps, so I'd expect to see both enum keys and enum values in this list.

Reply:
I saw this on the intrewebs. Tests would have failed if it didn't work

https://github.com/microsoft/TypeScript/issues/17198#issuecomment-448095764

Reply:
Hmm, now that comment seems funky.

Reply:
In that comment they do an additional filter though.

Reply:
```console
 ❯ ts-node                                                                                                 [23:44:57]
> enum Op {
... GET = 'GetItem',
... }
undefined
> Object.values(Op)
[ 'GetItem' ]
> 
```